src/extension/geofence/geofence-server/src/main/java/org/geoserver/geofence/server/rest/RulesRestController.java

```diff
@@ -167,6 +175,10 @@ public JaxbRuleList count(
             @RequestParam(value = "instanceId", required = false) Long instanceId,
             @RequestParam(value = "instanceName", required = false) String instanceName,
             @RequestParam(value = "instanceAny", required = false) Boolean instanceDefault,
+            @RequestParam(value = "ipaddress", required = false) String ipaddress,
+            @RequestParam(value = "ipaddressAny", required = false) Boolean ipaddressDefault,
+            @RequestParam(value = "date", required = false) String date,
```

Comment:
So is this a date range, always? Like yyyy-mm-dd/yyyy-mm-dd? If so I would recommend calling this a "dateRange"

Reply:
These are the params for getting the Rules, so, again, these are filtering params.